pkg/scalers/azure/azure_app_insights.go

```diff
@@ -0,0 +1,145 @@
+package azure
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+
+	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
minor nit: would you mind moving `logf "sigs.k8s.io/controller-runtime/pkg/log"` to the block above, next to the Azure deps.

Reply:
should be all set now